static/app/components/performance/vitalsAlert/vitalsAlertCTA.tsx

```diff
@@ -70,18 +75,34 @@ export default function VitalsAlertCTA({data, dismissAlert}: Props) {
   // persist to dismiss alert
   const api = useApi({persistInFlight: true});
   const vital = getWorstVital(data);
-  if (!vital) {
-    return null;
-  }
-  const ourValue = data[vital];
-  if (!ourValue) {
-    return null;
-  }
-  const sentryDiff = getRelativeDiff(ourValue, SENTRY_CUSTOMERS[vital]);
-  const industryDiff = getRelativeDiff(ourValue, INDUSTRY_STANDARDS[vital]);
+  const ourValue = vital ? data[vital] : 0;
+  const sentryDiff = vital ? getRelativeDiff(ourValue, SENTRY_CUSTOMERS[vital]) : 0;
+  const industryDiff = vital ? getRelativeDiff(ourValue, INDUSTRY_STANDARDS[vital]) : 0;
+
+  const hasGlobalViews = organization.features.includes('global-views');
+  // find the project that has the most events of the same type
+  const bestProjectData = vital
+    ? maxBy(data.projectData, item => {
+        const parameterName = getCountParameterName(vital);
+        return item[parameterName];
+      })
+    : null;
+
+  const showVitalsAlert = () => {
+    // check if we have the vital and the count is at least at the min
+    if (!vital || ourValue < MIN_VITAL_COUNT_FOR_DISPLAY) {
+      return false;
+    }
+    // if worst vital is better than Sentry users, we shouldn't show this alert
+    if (sentryDiff < 0) {
+      return false;
+    }
+    // must have either global views enabled or we can pick a specific project
+    return hasGlobalViews || bestProjectData;
+  };
 
-  // if worst vital is better than Sentry users, we shouldn't show this alert
-  if (sentryDiff < 0) {
+  // TODO: log experiment
+  if (!vital || !showVitalsAlert()) {
```

Comment:
when !vital, showVitalsAlert will always return false right? Don't think you need this additional check here.

Reply:
@Neo-Zhixing Correct, but TS doesn't know that. I'm doing that here so TS knows `vital` isn't falsy.